docs/source/getting_started/installation.md

```diff
@@ -26,15 +26,18 @@ Note: JupyterLab currently requires a re-build after installing any extension.
 * [Node.js 12+](https://nodejs.org/en/)
 * [Python 3.x](https://www.python.org/downloads/)
 
-Note: Elyra 1.0.0 and above require JupyterLab 2.x.
-Note: Elyra 2.0.0 and above require JupyterLab 3.x.
+
+JupyterLab dependencies:
+ - Elyra >= 2.0.0 requires JupyterLab 3.x
+ - Elyra < 2.0.0 requires JupyterLab 2.x
+ - Elyra == 0.10.x requires JupyterLab 1.x
 
 ### pip
 
-If you use `pip`, install Elyra with:
+If you use `pip`, install Elyra 2.x with:
 
 ```bash
-pip install --upgrade elyra && jupyter lab build
+pip install --upgrade "elyra>=2.0.1" && jupyter lab build
 ```
```

Comment:
why are we using `>=2.0.1`? just `elyra` will default to the latest.... and having the version will require making sure the release script is updating it for every release.

Reply:
I had the same change in the readme